packages/frontend/src/main/main.ts

```diff
@@ -331,19 +323,15 @@ app.on('ready', async () => {
   ipcMain.on('start-waggle', async () => {
     await waggleProcess?.connectionsManager.closeAllServices()
     await waggleProcess?.dataServer.close()
-    waggleProcess = await runWaggle(mainWindow.webContents)
+    waggleProcess = await runWaggle(mainWindow.webContents, appDataPath)
   })
 })
 
 app.setAsDefaultProtocolClient('quiet')
 
-app.on('before-quit', () => {
-  if (browserWidth && browserHeight) {
-    electronStore.set('windowSize', {
```

Comment:
We don't need to set window size anymore?

Reply:
We don't have electron store anymore, so on app start we start with default window size

Reply:
We will probably add 'user settings' in the future